test/service/test_response.py

```diff
@@ -1225,13 +1225,12 @@ def test_sample(self):
                         for sample in hit['samples']:
                             sample_entity_type = sample['sampleEntityType']
                             for key, val in sample.items():
-                                if key in ['sampleEntityType','effectiveOrgan']:
-                                    continue
-                                if isinstance(val, list):
-                                    for one_val in val:
-                                        self.assertIn(one_val, hit[sample_entity_type][0][key])
-                                else:
-                                    self.assertIn(val, hit[sample_entity_type][0][key])
+                                if key not in ['sampleEntityType', 'effectiveOrgan']:
```

Comment:
I'd default to using a tuple. It may not make a significant difference performance-wise but it emphasizes the immutability of the literal value.